### DISCUSSION TITLE
Github copilot stop working

### DISCUSSION THREAD

**Discussion by TheDeku** [community/community#40078] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hello community,

Today when I check my copilot status says suscription as ended. I can pay the suscription, but I have Student account and I'm not able to pay it because in the setting page for my status account is free, So I think it is a bug.

Can you help me to fix this please I use that service a lot

**Comment by SilasRodrigues19** [comment]
@TheDeku Hi bro.

When you access [this link](https://github.com/settings/billing) does it appear like this as the image below?

![image](https://user-images.githubusercontent.com/47532703/204016658-b3e88573-c3f1-4a73-9047-590b641c4426.png)

**Reply by SilasRodrigues19** [reply]
In my list of services provided by the GitHub Student Developer Pack, Copilot and Canvas are no longer showing.

**Reply by TheDeku** [reply]
Yes I have that message in the suscription list

**Comment by jfullstackdev** [comment, marked as answer]
I'm seeing a lot of these things, you can check this out since one GitHub Staff already responded there

https://github.com/orgs/community/discussions/40037
